static/sw.js

```diff
@@ -12,28 +20,78 @@ this.addEventListener("install", event => {
       console.log("Opened cache");
       return cache.addAll(URLS);
     })
-    .catch(err => { 
+    .catch(err => {
       console.log(err);
       throw err;
     })
   );
 });
 
+this.addEventListener("sync", event => {
+  if(event.tag == 'save-results') {
```

Comment:
❗️
Пройдись линтером по файлику
```suggestion
  if (event.tag == 'save-results') {
```